src/image-question/components/app.tsx

```diff
@@ -19,6 +18,8 @@ export interface IAuthoredState extends IAuthoringInteractiveMetadata {
   answerType: string;
   defaultAnswer: string;
   modalSupported?: boolean;
+  useSnapshot?: boolean;
```

Comment:
This used to be a source property instead of a boolean flag. 
At least it was in the UI: 
![image](https://user-images.githubusercontent.com/86016/94951137-c376d700-04b1-11eb-8697-74c0581c996b.png)

However making it a set of flags does seem more flexible. It would let an author setup a image question that lets a user take a snapshot or upload or just start drawing.

Reply:
Okay, I'll keep that in mind while adding upcoming features and change this field.